server/controllers/userController.js

```diff
@@ -11,9 +12,40 @@ class UserController {
       });
   }
 
+  static login(req, res) {
+    req.checkBody('email', 'Email field is required').notEmpty().trim();
```

Comment:
Extract this validation to another file. It is a good practice to have a validation file that is responsible for checking the users input before bringing it into the controller.

Reply:
Just got it done.